styles.css

```diff
@@ -264,3 +264,85 @@ button:disabled {
   color: #396df2;
   width: 50%;
 }
+
+/* contact form styling */
+
+.contact-form {
+  display: flex;
+  flex-direction: column;
+  border-top-left-radius: 80px;
+  background: #6070ff;
+  align-items: center;
+  width: 22vh 8vw;
+}
+
+.contact-me {
+  margin-top: 103px;
+  justify-content: center;
+  font-size: 40px;
+  font-weight: 700;
+  color: #fff;
+}
+
+.contact-form p {
+  font-size: 20px;
+  font-weight: 400;
+  text-align: center;
+  color: #fff;
+}
+
+.contact-item {
+  justify-content: center;
+  margin-top: 5%;
+  margin-bottom: 5%;
+}
+
+.contact-form form {
+  display: flex;
+  flex-direction: column;
+}
+
+.form-item {
+  margin-top: 5%;
+  margin-bottom: 5%;
+  border-radius: 8px;
+}
+
```

Comment:
- [ ] Please make sure to add a bit of `padding` to the following fields as given in the Figma template for getting a design close to the Figma template according to project requirements 🤓✌
- [ ] Also, kindly use a `border` for the following fields and reduce the space between them because of the above mentioned reason 👆 The border given is `border: 1px solid #CFD8DC;`

Your design | Given design
---| ---|
![image](https://github.com/ausiemass/portfolio-austin/assets/106119105/60138ced-08bb-45e6-a9ab-14a1bfeb00f9) | ![image](https://github.com/ausiemass/portfolio-austin/assets/106119105/ca5c03bc-e27d-4f90-a4ff-075c99a8f6ef)